docs/css/styles.css

```diff
@@ -136,23 +136,20 @@
   margin: 0 auto 1.5em; }
   @media screen and (min-width: 47em) {
     .c-AMElogo__color {
-      width: 14em; } }
+      max-width: 14em; } }
   @media screen and (min-width: 62.5em) {
     .c-AMElogo__color {
-      width: 18em; } }
+      max-width: 18em; } }
 
 .c-lion {
-  background-image: url("../images/lion.png"); }
-
-.inlinesvg .c-lion {
   background-image: none;
   width: 12.5em;
   position: absolute;
   bottom: 4em;
   left: calc(50% - 6.25em);
   z-index: 1; }
   @media (min-width: 62.5em) {
-    .inlinesvg .c-lion {
+    .c-lion {
```

Comment:
You could nest this media query inside the .c-lion class, and not have to call the class name a second time inside the query.